src/negotiation/execution_params.ts

```diff
@@ -26,6 +30,11 @@ export function validateExecutionParams(
   const now = moment().unix();
   const relativeAlphaExpiry = executionParams.alpha_expiry - now;
   const relativeBetaExpiry = executionParams.beta_expiry - now;
+
```

Comment:
Sorry that this feedback comes a bit late as it is not related to your PR but I have 3 points for the overall function `validateExecutionParams`:

1) the function validates if the two defined networks are of the same category and returns false otherwise: 
--> I personally would not enforce this without giving the user a possibility to disable this explicitly or at least stating so. 

2) imho the function is more about validating the expiries time and I would call it:
--> `validateExpiries`

3) There is a _bug_ on line 43: `relativeAlphaExpiry > 47 * 60 * 60 && relativeBetaExpiry > 23 * 60 * 60` 
--> it does not enforce the `alphaExpiry` to be greater then `betaExpiry` and that there is enough time in between. Imho this test should fail: 
```javascript
it("Invalidates mainnet execution parameters due to expiry", () => {
    const executionParams: ExecutionParams = {
      ...defaultExecutionParams(),
      alpha_expiry: moment().unix() + 24 * 60 * 60,
      beta_expiry: moment().unix() + 48 * 60 * 60,
    };

    expect(validateExecutionParams(executionParams)).toBeFalsy();
  });
```


--- 

Point 1 & 2 are not that important, point 3 should be fixed though. 
Maybe for now this would be good enough: 
```
relativeBetaExpiry > 23 * 60 * 60 && relativeAlphaExpiry > 2 * relativeBetaExpiry
```

Reply:
👍
I'd either invite @da-kami to add that to the feedback we want. What do dev think about expiries?
Yes ultimately we want this to be flexible but my understanding is that for now, we set it.